mac.sh

```diff
@@ -150,8 +150,11 @@ install_latest_ruby() {
   rbenv shell "$ruby_version"
 
   gem update --system
+}
 
-  gem install bundler --version 1.9.9
+install_bundler_1_17_3() {
+  gem install bundler --version 1.17.3
+  rbenv rehash
```

Comment:
rehash after installing an executable. Mimics the functionality in the `gem_install_or_update` function above